How identity works in Camunda page (#9577)
camunda/camunda-docs: docs/self-managed/components/identity/how-identity-works.md

---
id: how-identity-works
title: "How identity works in Camunda"
sidebar_label: "How identity works"
description: "Camunda Self-Managed uses two separate identity subsystems. Learn which one controls what, and which one you need to configure."
---

import DocCardList from '@theme/DocCardList';

Camunda Self-Managed uses two separate identity subsystems.

> Understanding which one controls what will save you from a common class of misconfiguration — connecting your identity provider (IdP) to one system but not the other, then wondering why some components authenticate but others don't.

Reviewer: ```suggestion
Understanding which subsystem controls what helps you avoid a common misconfiguration — connecting your identity provider (IdP) to one system but not the other, so some components authenticate while others don't.
```

For the full breakdown, see [Admin vs Management Identity](/self-managed/reference-architecture/reference-architecture.md#admin-vs-management-identity).

In most deployments, both subsystems share the same IdP — you create a separate OIDC application registration for each subsystem, but manage users in one place at the IdP level.

## Background: the 8.8 identity split

Before Camunda 8.8, [Management Identity](/self-managed/components/management-identity/overview.md) (then called just "Identity") managed access for every component, including Zeebe, Operate, and Tasklist. That release split it in two: the [Orchestration Cluster](/self-managed/reference-architecture/reference-architecture.md#orchestration-cluster) began managing its own authentication and authorization internally, through [Admin](/self-managed/components/orchestration-cluster/admin/overview.md) (formerly called Orchestration Cluster Identity). See [Identity, authentication, and authorization](/reference/announcements-release-notes/880/whats-new-in-88.md#identity) for the full migration details.

> :::note Manage roles in Admin after upgrading

Reviewer: Feels like this info is more inline text than admonition

> This split is the root cause behind one of the most common sources of post-upgrade confusion: role and authorization changes made in Console no longer affect an already-migrated cluster. Existing roles and authorizations are migrated automatically during the upgrade, but from that point on, the cluster's authorizations live in Admin — not in Management Identity or Console. If you change a role in Console expecting it to affect Operate or Tasklist access, it won't. Make that change in Admin instead.

Reviewer 2: Is this the right framing? (While I agree it's accurate, it comes across a bit negative)
Maybe something like 
"After upgrade, Admin becomes the single source of truth for the migrated cluster's roles and authorizations. During upgrade, existing roles and authorizations carry over automatically, so nothing needs to be re-created. After upgrade, you manage access to Operate and Tasklist in Admin — changes made in Console or Management Identity do not apply to the migrated cluster."

Author: Agreed 😄 Updated.
